spring-social-zotero/src/main/java/org/springframework/social/zotero/api/impl/GroupsTemplate.java

```diff
@@ -358,6 +360,24 @@ private JsonNode createDataJson(Item item, List<String> ignoreFields, List<Strin
         }
     }
 
+    @Override
+    public List<ItemDeletionResponse> deleteMultipleItems(String groupId, List<String> citationKeys, Long citationVersion) throws ZoteroConnectionException {
+        List<ItemDeletionResponse> responses = new ArrayList<ItemDeletionResponse>();
+        for (int i = 0; i < citationKeys.size(); i += 50) {
+            List<String> subList = citationKeys.subList(i, Math.min(citationKeys.size(),i+50));
+            String citations = String.join(",", subList);
+            String url = String.format("groups/%s/%s", groupId, "items?itemKey=" + citations);
+
+            HttpHeaders headers = new HttpHeaders();
+            headers.set("If-Unmodified-Since-Version", citationVersion + "");
+
+            HttpEntity<JsonNode> dataHeader = new HttpEntity<JsonNode>(headers);
+            ResponseEntity<String> zoteroResponse = restTemplate.exchange(buildUri(url, false), HttpMethod.DELETE, dataHeader, new ParameterizedTypeReference<String>() {});
+            responses.add(ItemDeletionResponse.getStatusDescription(zoteroResponse.getStatusCode().value()));
```

Comment:
the method should also return for which ids the status code was returned, in case the order changes for some reason.